posthog/test/test_team_model.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+
+from ..models.team import Team
+
+
+class TestTeam(TestCase):
+    def test_team_str(self):
+        # Use the team.name by default
+        team: Team = Team.objects.create(name="The Mighty Library")
+        self.assertEqual(str(team), "The Mighty Library")
+
+        # If the team has no name:
+
+        # Team has one app_url
+        team = Team.objects.create(app_urls=["app.posthog.com"])
+        self.assertEqual(str(team), "app.posthog.com")
+
+        # Team has multiple app_urls
```

Comment:
I'd say these tests would be more readable if each case was a separate method (e.g. `def test_str_multiple_app_urls(self):`) instead of a comment. Probably a tiny bit slower though.

Reply:
Thanks @Twixes ! I'll keep it in mind for future changes, agreed! It might make more sense in some cases